### DISCUSSION TITLE
Is there any way to use partials with the web server?

### DISCUSSION THREAD

**Discussion by ghost** [serverpod/serverpod#579] [Q&A] [answered]
For example if I have a `<nav>`...`</nav>` element is there any way for me to reuse this code across different templates?

**Comment by vlidholt** [comment, marked as answer]
It is possible to nest templates. In that way a template can work almost like a widget in Flutter.

**Reply by ghost** [reply]
Thanks Viktor, that sounds perfect. Are there any examples of this? Would this be done in dart or would we need to us JS?

**Reply by vlidholt** [reply]
It's done in Dart on the server side, on the web side you can use whichever framework you prefer. E.g. you can use Dart, but most would probably use JavaScript.
